pkgs/applications/misc/wapanel/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub
+, meson, ninja, pkg-config
+, gtk3, glib, gtk-layer-shell, gdk-pixbuf, librsvg
+, wayland, wayland-protocols, wayland-scanner
+, libpulseaudio, xdg-utils
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wapanel";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Firstbober";
+    repo = "wapanel";
+    rev = version;
+    hash = "sha256-sqVH/XXFIEdB3b19eHj0gqN/hjEltxKisT/UAdMmmXA=";
+    fetchSubmodules = true;
+  };
+
+  # Change to `true` once ToruNiina/toml11 is packaged
```

Comment:
https://github.com/ToruNiina/toml11 seems to be straight forward to package and just cmake should already do it. Or did you run into problems?

Reply:
Sure, I had a derivation for it but forgot about merging it in this PR.